docs/quickstart.md

```diff
@@ -0,0 +1,407 @@
+
+```
+$ openapi lint petstore.yaml 
+No configurations were defined in extends -- using built in recommended configuration by default.
+
+validating petstore.yaml...
+petstore.yaml: validated in 44ms
+
+Woohoo! Your OpenAPI definition is valid. 🎉
+```
+
+Congratulations! You did a great job and now your OpenAPI definition is valid!
+
+### split
+
+With this command, your OpenAPI definition file will be divided into constituent parts to follow the multi-file approach. `split` does the opposite of the [`bundle`](#bundle) command.
+
+In your terminal, type the following:
+
+```
+$ openapi split petstore.yaml --outDir openapi
+```
+
+You should get the following response:
+
+```
+$ openapi split petstore.yaml --outDir openapi
+Document: petstore.yaml is successfully split
+ and all related files are saved to the directory: openapi 
+
+petstore.yaml: split processed in 44ms
+```
+
+Cool, split completed. But what actually happened?
+
+Open the `openapi` directory. See? That's where the magic took place:
+
+```
+.
+├── components
+│   └── schemas
+│       ├── Error.yaml
+│       ├── Pet.yaml
+│       └── Pets.yaml
+├── paths
+│   ├── pets.yaml
+│   └── pets@{petId}.yaml
+└── openapi.yaml
+```
+
+With the `split` command, you can easily switch to a multi-file structure in a blink of an eye even if your definition file became extremely lengthy.
+
+### bundle
+
+With this command, you can merge standalone files back into a single definition file. `bundle` does the opposite of the [`split`](#split) command.
+
+Let's bundle the definition file that we split earlier.
+
+In your terminal, type the following:
+
+```
+$ openapi bundle openapi/openapi.yaml --output bundled.yaml
+```
+
+You should get the following response:
+
+```
+$ openapi bundle openapi/openapi.yaml --output bundled.yaml
+bundling openapi/openapi.yaml...
+📦 Created a bundle for openapi/openapi.yaml at bundled.yaml 46ms.
+```
+
+You should now see a new `bundled.yaml` file as a result of the `bundle` command completion. Open this file in your text editor or IDE and compare it with the linted definition file. You can see that both files are the same which means that the `bundle` command executed flawlessly.
+
+### preview-docs
+
+With this command, you can run a server to preview your OpenAPI definition file in real-time mode.
+
+In your terminal, type the following:
+
+```
+$ openapi preview-docs petstore.yaml 
+```
+
+You should get the following response:
+
+```
+$ openapi preview-docs petstore.yaml 
+Using Redoc community edition.
+Login with openapi-cli login or use an enterprise license key to preview with the premium docs.
+
+
+  🔎  Preview server running at http://127.0.0.1:8080
+
+Bundling...
+
+
+  👀  Watching petstore.yaml and all related resources for changes
+
+Created a bundle for petstore.yaml successfully
+```
+
+Navigate to `http://127.0.0.1:8080` and check that your definition file has been served successfully:
+
+![preview-docs](./images/preview-docs.png)
```

Comment:
I wonder if it makes sense to make this image smaller (more cropped possibly). This is not a showstopper for me, so I'm good with the guide overall.